sdcm/nemesis.py

```diff
@@ -34,7 +34,8 @@
 from invoke import UnexpectedExit  # pylint: disable=import-error
 from cassandra import ConsistencyLevel
 
-from sdcm.cluster import SCYLLA_YAML_PATH, NodeSetupTimeout, NodeSetupFailed, ClusterNodesNotReady
+from sdcm.paths import SCYLLA_YAML_PATH
+from sdcm.cluster import NodeSetupTimeout, NodeSetupFailed, ClusterNodesNotReady
 from sdcm.cluster import NodeStayInClusterAfterDecommission
```

Comment:
Following imports look strange:

```
from sdcm.cluster import NodeSetupTimeout, NodeSetupFailed, ClusterNodesNotReady
from sdcm.cluster import NodeStayInClusterAfterDecommission
```

I propose to make such imports in the following way:

```
from sdcm.cluster import (
    ClusterNodesNotReady,
    NodeSetupFailed,
    NodeSetupTimeout,
    NodeStayInClusterAfterDecommission,
)
```

It becomes obvious that we import from the same module and can update less code lines in future.

Reply:
Sure. I think it appears after last rebase. Will fix.

Reply:
fixed

Reply:
fixed